python/ray/serve/_private/constants.py

```diff
@@ -191,3 +191,6 @@ class ServeHandleType(str, Enum):
 
 # Serve HTTP request header key for routing requests.
 SERVE_MULTIPLEXED_MODEL_ID = "serve_multiplexed_model_id"
```

Comment:
oh, should this one be all-caps as well?

Reply:
I think headers are typically case-insensitive, but might as well be consistent...

Reply:
hmm, good point, I can push a fix in the master (since headers are case insensitive, I will not do cherry-pick in this case)